web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/dto/behandling/BehandlingsresultatDto.java

```diff
@@ -29,6 +29,8 @@ public class BehandlingsresultatDto {
     private List<KonsekvensForYtelsen> konsekvenserForYtelsen;
     @JsonProperty("vedtaksbrev")
     private Vedtaksbrev vedtaksbrev;
+    @JsonProperty("vedtaksbrevBlirProdusertOgSendtUt")
```

Comment:
Har du ikke allerede dette i Vedtaksbrev rett over ? (AUTO/FRI/INGEN)
Evt vurdere enum for forteller årsak SEND/ETTERKLAGE/INGENENDRING